app/src/org/commcare/adapters/EntityListAdapter.java

```diff
@@ -261,7 +261,9 @@ private View getEntityView(Entity<TreeReference> entity, EntityView emv, int pos
                     getCalloutDataForEntity(entity));
         } else {
             emv.setSearchTerms(currentSearchTerms);
-            emv.setExtraData(detail.getCallout().getResponseDetailField(), getCalloutDataForEntity(entity));
+            if (detail.getCallout() != null) {
```

Comment:
probably also need to check if `detail.getCallout().getResponseDetailField()` is `null`